baby-bear/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [features]
-nightly-features = []
+nightly-features = ["p3-monty-31/nightly-features"]
```

Comment:
If you are using the baby-bear crate (just as an example running the tests) and you are specifying the nightly-feature on a cpu supporting avx512, if such a feature is not propagated to the p3-monty-31 crate it will generate a compilation issue because `baby-bear/src/x86_64_avx512/packing.rs:4:19 `requires `MontyParametersAVX512`. 

You can test it on a machine supporting avx512, reverting such change in baby-bear/Cargo.toml and executing `Plonky3/baby-bear$ RUSTFLAGS=-Ctarget-cpu=native cargo +nightly-2024-09-23 test --release --features nightly-features`
You should get the following compilation error:
```
error[E0432]: unresolved imports `p3_monty_31::MontyParametersAVX512`, `p3_monty_31::PackedMontyField31AVX512`
 --> baby-bear/src/x86_64_avx512/packing.rs:4:19

```